src/com/tt/APIUtils.java

```diff
@@ -49,12 +49,12 @@
         return getAnyKeyValueAsString(jsonObject, responseKey);
     }
 
-//    public static void main(String[] args) {
-//        String str = "{\"test\":\"ing\",\"weather\":[{\"id\":711,\"main\":\"Smoke\",\"description\":\"smoke\",\"icon\":\"50n\"}]}";
-//        JSONObject json = new JSONObject(str);
-//
-//        System.out.println(Arrays.toString(getAnyKeyValueAsString(json, new String[]{"weather.0.icon"})));
-//    }
+   public static void main(String[] args) {
+       String str = "{\"test\":\"ing\",\"weather\":[{\"id\":711,\"main\":\"Smoke\",\"description\":\"smoke\",\"icon\":\"50n\"}]}";
+       JSONObject json = new JSONObject(str);
+
+       System.out.println(Arrays.toString(getAnyKeyValueAsString(json, new String[]{"weather.0.icon"})));
+   }
```

Comment:
This was meant only for debugging purpose. You can comment or remove the main method from APIUtils class. Actually there should be junit tests for this class but we'll add them later.